test/e2e/far_e2e_test.go

```diff
@@ -277,3 +310,18 @@ func wasNodeRebooted(nodeName string, nodeBootTimeBefore time.Time) {
 
 	log.Info("successful reboot", "node", nodeName, "offset between last boot", nodeBootTimeAfter.Sub(nodeBootTimeBefore), "new boot time", nodeBootTimeAfter)
 }
+
+// remediateNode run three functions to verify whether the node was remediated
+func remediateNode(nodeName, logString string, nodeBootTimeBefore time.Time) {
+	By("Adding FAR NoExecute taint")
+	wasFarTaintAdded(nodeName)
+
+	By("Executing the FA command and receive success response")
+	// TODO: When reboot is running only once and it is running on FAR node, then FAR pod will
```

Comment:
what's the plan for this TODO? This sounds like this test will fail regularly?

Reply:
>When reboot is running only once

ATM reconcile can be triggered for some reasons, e.g. when far pod was evicted.
When it is evicted, e.g. from FAR taint, then it will run the reboot again and then a success message should appear. But when we will resolve this problem (with status), then we should skip the check for the success message when we taint the node which had the FAR pod.